### DISCUSSION TITLE
googleapis.com

### DISCUSSION THREAD

**Discussion by ERa777github** [Anonym-tsk/tpws-keenetic#14] [Обсуждения]
Вопрос такой, зачем добавлен в юзер лист googleapis.com ? Если он в листе то не работает гугл плей.

**Comment by Anonym-tsk** [comment]
Приложение ютуба им пользуется. Если вам не нужен, удалите

**Reply by ERa777github** [reply]
причем тут вам не нужно. ну введите всё домены подряд по дефаулту кому надо тот будет удалят если у него что то не работает.  Без гуглапи что ютуб не работает? работает. А то что другой софт не работает это да. У кого то приложения не обновляются автоматом с гугла а он и не в курсе что сломал установкой сие утилиты.

**Reply by Anonym-tsk** [reply]
Приложения не обновляются потому что QUIC не заблокирован. Этот конкретный домен здесь вообще никак

**Comment by bysashka** [comment]
tpws и antizapret на OpenVpn могут совместно работать? Или лучше оставить что-то одно?

**Reply by Anonym-tsk** [reply]
Могут
